### ISSUE TITLE
Linux

### ISSUE THREAD

**Issue by pahosler** [campDevs/DiscordBot#18]
add a Linux command:
+Linux distros - list top 5 or 10 distros
+Linux get <distro> - will post a torrent magnet link
+Linux help <command> - post brief help
...<more possibilities for this>

**Comment by jodylecompte** [comment]
Regarding item 3,  I found this library that I think would be excellent 
https://www.npmjs.com/package/tldr

I really like the idea of providing magnet links, Bittorrent is better for the maintainers (less bandwidth) and in cases of major distros like Ubuntu the torrent download is usually faster.

**Comment by glynnw** [comment]
Yeah this is a cool idea and made me think of possibly adding the arch wiki as a place to search for help pertaining to a specific linux topic.
